PhysicsTools/PatAlgos/python/slimming/puppiForMET_cff.py

```diff
@@ -18,25 +18,26 @@ def makePuppies( process ):
 
 
     process.puppiNoLep = process.puppi.clone()
-    process.puppiNoLep.candName = cms.InputTag('pfNoLepPUPPI') 
-
-    process.puppiMerged = cms.EDProducer("CandViewMerger",src = cms.VInputTag( 'puppiNoLep','pfLeptonsPUPPET'))
+    process.puppiNoLep.candName = cms.InputTag('particleFlow')
+    process.puppiNoLep.useWeightsNoLep = cms.bool(True)
+    
+#    process.puppiMerged = cms.EDProducer("CandViewMerger",src = cms.VInputTag( 'puppiNoLep','pfLeptonsPUPPET'))
```

Comment:
is this commented out code needed?
Please remove or add comments inline in the code why the commented out block is relevant